src/targets/gpu/device/include/migraphx/gpu/device/launch.hpp

```diff
@@ -76,7 +76,7 @@ MIGRAPHX_DEVICE_CONSTEXPR auto gs_invoke(F&& f, index_int i, index) -> decltype(
 inline auto gs_launch(hipStream_t stream, index_int n, index_int local = 1024)
 {
     index_int groups  = (n + local - 1) / local;
-    index_int nglobal = std::min<index_int>(256, groups) * local;
+    index_int nglobal = std::min<index_int>(1048576, groups) * local;
```

Comment:
Was this tested on vega20? This also affects all kernels not just softmax. So how does this affect the overall performance of models(not just bert)?

Reply:
Can we change this value to something more generic like 1,000,000 instead of a random number?

Reply:
changed to 1 billion.

Reply:
@kahmed10 Do you have other comments for this PR?